app/src/test/java/org/mozilla/focus/web/DownloadTest.java

```diff
@@ -24,14 +24,16 @@ public void testGetters() {
                 "Content-Disposition: attachment; filename=\"filename.png\"",
                 "image/png",
                 1024,
-                Environment.DIRECTORY_DOWNLOADS);
+                Environment.DIRECTORY_DOWNLOADS,
+                "filename.png");
```

Comment:
\[PMD\] INFO: The String literal "filename\.png" appears 4 times in this file; the first occurrence is on line 28